spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/VersionedAppRegistryController.java

```diff
@@ -284,6 +289,26 @@ else if (!CollectionUtils.isEmpty(apps)) {
 				assembler);
 	}
 
+	/**
+	 * Unregisters the list of applications provided via request parameter.
+	 * @param request - Contains a list of applications to be removed
+	 * @return A response with the status of each requested application with either success or error
+	 */
+	@RequestMapping(method = RequestMethod.POST, value = "/bulk/unregister")
+	@ResponseStatus(HttpStatus.OK)
+	public BulkAppRegisterResponse bulkUnregister(@RequestBody BulkAppRegisterRequest request) {
```

Comment:
Looks like this method could go into a common controller and/or service layer method that is shared between skipper and classic modes to reduce code duplication.